dash-sdk-android/src/main/cpp/CMakeLists.txt

```diff
@@ -49,16 +49,23 @@ add_custom_command(
 add_custom_target(BuildRustLib ALL
     DEPENDS ${RUST_LIB_OUTPUT_PATH}
 )
+set_target_properties(BuildRustLib PROPERTIES
+    LOCATION ${RUST_LIB_OUTPUT_PATH}
+)
 
 # Java
 set(JAVA_AWT_LIBRARY NotNeeded)
 set(JAVA_JVM_LIBRARY NotNeeded)
 set(JAVA_INCLUDE_PATH2 NotNeeded)
 set(JAVA_AWT_INCLUDE_PATH NotNeeded)
 find_package(JNI REQUIRED)
+# required for android log
+# find_library(LOG_LIB log)
+# find_library(ANDROID_LIB android)
```

Comment:
We will need this later for when we can get the rust tracer to connect to the android loggers.